src/platforms/react-native/usage/event-information.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "Event Information"
+description: "Learn about the information we attach onto events."
+---
+
+The Sentry React Native SDK will automatically add information onto your event to make it easier to debug and understand your app.
+
+## Event Origin and Environment Tags
+
+The event origin tag tells you which layer of your app the event originated from.
+
+The possible values for `event.origin` are:
```

Comment:
Can we put `event.origin` and `event.environment` and it’s possible values in the develop docs?

Reply:
Agreed, so we can align the values across the SDKs